runtime/doc/syntax.txt

```diff
@@ -4665,7 +4665,8 @@ cterm={attr-list}			*attr-list* *highlight-cterm* *E418*
 
 	Note that "bold" can be used here and by using a bold font.  They
 	have the same effect.
-	If running in a terminal, "undercurl" acts as an alias for "underline".
+	"undercurl" falls back to "underline" in a terminal that does not
+        support it.
```

Comment:
indent should be `<tab>` on this line, like the one above it. Wording suggestion: